src/order/rel_classes.lean

```diff
@@ -74,6 +74,14 @@ protected theorem is_asymm.is_irrefl [is_asymm α r] : is_irrefl α r :=
 protected theorem is_total.is_trichotomous (r) [is_total α r] : is_trichotomous α r :=
 ⟨λ a b, or.left_comm.1 (or.inr $ total_of r a b)⟩
 
+instance is_refl_of_irrefl {α : Type u} (r : α → α → Prop) [is_irrefl α r] :
+  is_refl α (λ x y, ¬ r y x) :=
```

Comment:
Why did you swap the arguments here, and not
```suggestion
  is_refl α (λ x y, ¬ r x y) :=
```

Reply:
Well, that's actually two theorems that are both true. I can add those instances as well.

Reply:
I should have commented this in #13411, oops